### DISCUSSION TITLE
No media to process

### DISCUSSION THREAD

**Discussion by czarnynight** [aclap-dev/video-downloadhelper#534] [Help and Q&A]
### Your question

after update to 9.0 on chrome and opera gx extension is not seeing any videos on new ui. after changing to old UI it is able to correctly detect all videos but it's unable to download any of them give them all .htm extension and failing (regardless if i keep .htm or change to mp4). yesterday when i downloaded on v8 all worked fine.

### Addon & CoApp version and details

version: 9.0.1.13
target: google
channel: stable
lang: pl
coapp: "checking"
license: {"checking":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36 OPR/109.0.0.0
{
    "used_sidebar_button": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 6
}

**Comment by paulrouget** [comment]
Do you mind trying this:

- Open Video DownloadHelper popup
- Right click inside the popup
- Click "Inspect" - a window will open
- Click on "Console" at the very top
- Tell me if it shows any errors

**Reply by czarnynight** [reply]
main.js:31 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'setOptions')
    at sl (main.js:31:24771)
    at Bf (main.js:31:24696)
    at main.js:38:2877
main.js:50 Couldn't not parse mimetype application/octet-stream
(anonymous) @ main.js:50
main.js:50 Couldn't not parse mimetype application/octet-stream
(anonymous) @ main.js:50
main.js:50 Couldn't not parse mimetype application/octet-stream
(anonymous) @ main.js:50
main.js:50 Couldn't not parse mimetype image/gif
(anonymous) @ main.js:50
main.js:50 Couldn't not parse mimetype image/gif
(anonymous) @ main.js:50
main.js:50 Couldn't not parse mimetype image/gif
(anonymous) @ main.js:50

**Reply by paulrouget** [reply]
Thank you! I know what's going on. Gonna fix that as soon as possible :)

**Reply by czarnynight** [reply]
also is there a way to change language ? i prefer using eng to pl. v8 and earlier were in eng. this is in pl from whatever reason.....

**Reply by paulrouget** [reply]
Is your browser in PL ?

**Reply by czarnynight** [reply]
chrome in eng, opera gx in pl. and i have pl on both, like it's checking system and not browser language. but it's secondary thing. primary is unable to use extension.

**Reply by paulrouget** [reply]
For the extension, we are publishing a version with a fix right now. Will take a little of time for Google to update your extension.

**Reply by czarnynight** [reply]
any eta for it?

**Reply by paulrouget** [reply]
Hoping it's gonna update today.

**Reply by czarnynight** [reply]
ok. updated to 9.0.1.18
it now sees videos but is unable to download any, and i'm not talking about streaming ones that req app. when i click download i get 
filename.htm and it fail download.
![what it sees at](https://github.com/aclap-dev/video-downloadhelper/assets/170521281/7acd685d-c58a-4bf5-9708-87d0d74bf519)

tried few other places.
tried now on same player same file just different res. 480p fail, 720p first was htm (fail) then was mp4 (succeeded), 1080p worked from get go, 4k downloaded

**Reply by paulrouget** [reply]
What website? Please report the whole error message.

**Reply by czarnynight** [reply]
website or player name ?
player CDA
main.js:35 No download item found
kw @ main.js:35
main.js:53 Couldn't not parse mimetype text/html
(anonymous) @ main.js:53
main.js:53 Couldn't not parse mimetype text/html
(anonymous) @ main.js:53
main.js:35 No download item found
kw @ main.js:35
main.js:53 Couldn't not parse mimetype text/html
(anonymous) @ main.js:53
main.js:53 Couldn't not parse mimetype text/html
(anonymous) @ main.js:53
i still have places to check if it works there but i mostly watch things that are on this player/platform

**Reply by paulrouget** [reply]
What website are you downloading from? I can't debug without the URL of the website.

**Reply by czarnynight** [reply]
one of problematic links
https://anime-odcinki.pl/anime/oban-star-racers/1/ 
player cda.
rest req app

**Reply by paulrouget** [reply]
> https://anime-odcinki.pl/anime/oban-star-racers/1/

this seems to work for me. what is not working exactly? Copy the error you're seeing please.

**Reply by GrampaWildWilly** [reply]
Yeah, it worked for me, too.  Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.  It's only 27 seconds long according to the player in the web page, & that's what I got from VDH.

**Reply by czarnynight** [reply]
opera gx 109.0.5097.100
main.js:35 No download item found
kw @ main.js:35
favicon.ico:1 
        
        
       Failed to load resource: the server responded with a status of 404 ()
favicon.ico:1 
        
        
       Failed to load resource: the server responded with a status of 404 ()

**Reply by czarnynight** [reply]
version: 9.0.1.18
target: google
channel: stable
lang: pl
coapp: {"error":"Specified native messaging host not found.","found":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36 OPR/109.0.0.0
{
    "used_sidebar_button": true,
    "show_success_notification": false,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_legacy_ui": false,
    "use_sidebar": true,
    "open_count_store": 4
}
in case you need it

**Comment by paulrouget** [comment]
There's an issue with your CoApp. You installed the latest version? Make sure your antivirus is not messing up with it.

**Reply by czarnynight** [reply]
no app and I will not install it. also if it req app it says so, those files worked fine without it in the past (may 12th)
those are not streaming files that would req app.
![image](https://github.com/aclap-dev/video-downloadhelper/assets/170521281/f83b8b75-6c1a-4001-9217-c3c11c98ed75)
example what i would see if i clicked on "streaming" type, and it's not my problem. streaming ones ik and i do not care about at all. i care about those that worked and now give me middle finger in the form of .htm and downloads fail on those in 0.01s after clicking enter

**Reply by paulrouget** [reply]
ah ok!

In the settings, can you enable Show Low Quality Media? Does it help?

<img width="336" alt="Screenshot 2024-05-26 at 09 15 08" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/8c8d8ae7-a71c-4bf1-9e81-9692be33cef7">

**Reply by czarnynight** [reply]
i can see them all the time just not download.
also i noticed something, when i hit "copy url"
![image](https://github.com/aclap-dev/video-downloadhelper/assets/170521281/782687e8-71a9-4f30-9873-0be204bc467c)
480p
https://vcze103.cda.pl/e/b/d/E0lOLij-I5HrRNCLW_vnuw/339b/Zmh3MkJNTU5VanAvZUxvZFZxRlk/1716741023/lqcc3bd7aa5a3e9a138ef69357022a9407.mp4
1080p
https://vcze602.cda.pl/h/b/b/24/na/ZQ7aBZAvFZ8JJN2W9BoTYA/33/1716741122/hdcc3bd7aa5a3e9a138ef69357022a9407.mp4
it's unable to find 480p file. in 1080p it shows  video. maybe fault is inability to find it so it try to download html and not video
